app/views/devise/sessions/new.html.erb

```diff
@@ -0,0 +1,29 @@
+<div class="card">
+  <h2 class="card-header">Log In</h2>
+  <div class="card-body">
+    <%= form_for(resource, as: resource_name, url: session_path(resource_name)) do |f| %>
+      <div class="field form-group">
+        <% f.label :email %><br />
+        <%= f.email_field :email, autofocus: true, autocomplete: "email", class: "form-control", placeholder: "Email" %>
+      </div>
+
+      <div class="field form-group mb-3">
+        <% f.label :password %><br />
+        <%= f.password_field :password, autocomplete: "current-password",  class: "form-control", placeholder: "Password" %>
+      </div>
+
+      <% if devise_mapping.rememberable? %>
+        <div class="field form-group">
+          <%= f.check_box :remember_me %>
+          <%= f.label :remember_me %>
+        </div>
+      <% end %>
+
+      <div class="actions mt-5">
+        <%= f.submit "Log in", class: "btn btn-secondary" %>
+      </div>
```

Comment:
- [OPTIONAL ] Great job working on setting up the devise but I think it would be great if you worked on the wireframe to match the given wireframe so kindly make the button width the same as the input width as shown in the given design 🔥💪🏻And I am assign as optional because the previous reviewer didn't mention it

- [OPTIONAL ] I highly recommend making the signup and login page the same airframe to improve from user experience (UX)
- Your design
![image](https://github.com/Lawmsangi/blog-app/assets/58553711/090e3e7d-a5af-4086-9083-b762d299cd68)

- Given
![image](https://github.com/Lawmsangi/blog-app/assets/58553711/cf5940e7-8ae2-4abb-a2f6-360339caa4a6)